themes/rainbond/layouts/_default/dashboard.html

```diff
@@ -212,6 +212,40 @@ <h2>最佳实践</h2>
                 <div class="feature-guide">
                     <h2 class="tips">跟随这些文档实践</h2>
                     <div class="feature-box">
+                        <div class="right-box">
+                            <div class="txt">
+                                <h2>申请POC测试</h2>
+                                <p class="p1">20min+远程会议+讲解演示</p>
+                                <p>跟随架构师快速了解Rainbond项目</p>
+                            </div>
+                            <div class="btn-area visitor-show">
+                                <div class="btn book">
+                                    <div class="img">
+                                        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+                                            <g id="Support" fill="none" fill-rule="evenodd" stroke="none"
+                                                stroke-width="1">
+                                                <g id="Desktop-HD" fill="#202D40" fill-rule="nonzero"
+                                                    transform="translate(-1176 -1061)">
+                                                    <g id="read-this-documents" transform="translate(168 845)">
+                                                        <g id="download-ppt" transform="translate(843 63)">
+                                                            <g id="button-booking" transform="translate(151 142)">
+                                                                <g id="icons/presentation" transform="translate(14 11)">
+                                                                    <circle id="Oval" cx="4" cy="3" r="2" />
+                                                                    <path id="Path"
+                                                                        d="M10 8a1 1 0 0 0 0-2H3a2 2 0 0 0-2 2v8a1 1 0 0 0 2 0v-4h1v4a1 1 0 0 0 2 0V8h4z" />
+                                                                    <path id="Path"
+                                                                        d="M16 2H8v2h7v7H8v2h3.382l1.723 3.447a1 1 0 1 0 1.79-.894L13.618 13H16a1 1 0 0 0 1-1V3a1 1 0 0 0-1-1z" />
+                                                                </g>
+                                                            </g>
+                                                        </g>
+                                                    </g>
+                                                </g>
+                                            </g>
+                                        </svg>
+                                    </div>去申请
```

Comment:
How to jump？

Reply:
![image](https://user-images.githubusercontent.com/32888522/95847540-6c111a80-0d7f-11eb-9256-d3ef05dc42f7.png)